app/assets/javascripts/tests.js

```diff
@@ -1,31 +1,42 @@
-(function($) {
-  $(function () {
+// https://support.google.com/optimize/answer/9059383
```

Comment:
Looks like we can squash this commit in to e3657d0d4c889dd0c273286788423b630821070b?